src/vs/workbench/contrib/terminal/browser/media/shellIntegration.zsh

```diff
@@ -42,6 +42,14 @@ continuation_end() {
 	printf "\033]633;G\007"
 }
 
+rprompt_start() {
```

Comment:
I think this would be easier to read if it were 

```
right_prompt_start
RIGHT_PROMPT
```
etc.
scripts are already confusing enough IMO

Reply:
Can change the function name though